docs/userguides/accounts.md

```diff
@@ -0,0 +1,103 @@
+# Accounts
+
+Accounts in Ape come from [AccountAPI](../methoddocs/api.html#ape.api.accounts.AccountAPI) implementations (e.g. from plugins).
+There are typically two types of accounts:
+
+1. Test accounts
+2. Live network accounts
+
+Test accounts are useful for local network testing and debugging contracts.
+Live network accounts are for interacting with live blockchains and should be secured.
+
+To learn more about Ethereum accounts, see [the Ethereum documentation](https://ethereum.org/en/developers/docs/accounts/).
+
+## Test Accounts
+
+Ape ships with pytest fixtures to assist in writing your tests.
+Pre-funded test accounts are accessible via the [accounts fixture](./testing.html#accounts-fixture).
+
+```python
+def test_my_contract_method(accounts):
+    sender = accounts[0]
+    ...
+```
+
+To access the same prefunded accounts in your scripts or console, use the root `accounts` object and the [test_accounts](../methoddocs/managers.html#ape.managers.accounts.AccountManager.test_accounts) property:
+
+```python
+from ape import accounts
+
+sender = accounts.test_accounts[0]
+```
+
+You can configure your test accounts using your `ape-config.yaml` file:
+
+```yaml
+test:
+  mnemonic: test test test test test test test test test test test junk
+  number_of_accounts: 5
+```
+
```

Comment:
Add a note to NEVER use a seed phrase with funds on it here

Reply:
[cb557fe](https://github.com/ApeWorX/ape/pull/981/commits/cb557fe2f55e7b2f1d686278688fb0fc9e28149d)